src/vue/components/grammar.vue

```diff
@@ -1,20 +1,45 @@
 <template>
   <div class="alpheios-grammar">
-    <div class="alpheios-grammar__button--back-block">
-      <alph-tooltip :tooltipText="l10n.getText('TOOLTIP_BACK_TO_INDEX')" tooltipDirection="bottom-left">
-        <button @click="returnToIndex" class="alpheios-button-primary alpheios-svg-index"><back-icon /></button>
-      </alph-tooltip>
-    </div>
-    <div class="alpheios-grammar__frame-cont" v-if="this.hasGrammarResUrl">
-      <iframe :src="$store.state.app.grammarRes.url" class="alpheios-grammar__frame" scrolling="yes"></iframe>
+    <div class="alpheios-grammar__titles">
+        <h1 class="alpheios-panel__title">Browse Language Grammar</h1>
+      <div class="alpheios-grammar__block alpheios-clickable" 
+          :class="{open: !languageItem.collapsed}"
+           v-for="(languageItem, langIndex) in languageList" :key="langIndex">
+
+           <p class="alpheios-grammar__block__title"
+              @click="collapseLanguage(languageItem.languageCode)"
+           >{{ languageItem.title }}
```

Comment:
see also note above in the uicontroller - I think we need more advanced logic to determine which grammar is open based upon the current lexical query.

Reply:
Yes, they are both manually printed values in side `grammar.vue` component.
I believe that we should retrieve grammar title from the res-client.
So we need to make changes in res-client repo.
But as it would be refactored with new approach (used in CEDICT), I think it should be done res-client would be refactored.

Reply:
ok. i have entered this as a new issue in the res-client repo for now.